KTLibrary/KTLibrary.py

```diff
@@ -0,0 +1,38 @@
+import json
+from pathlib import Path
+from typing import Optional
+
+from KTLibrary.FormattedEncoder import FormattedEncoder
+
+
+class KTLibrary:
+    def __init__(self):
+        self.cache_file: Optional[dict] = None
+
+    def read_file(self, file_path: str):
+        self.cache_file = self.read_cache_file(file_path)
+
+    @staticmethod
+    def read_cache_file(file_path: str) -> dict:
+        # Read the cache file where we catch any exceptions if thrown and printed for the user to see.
+        try:
+            with open(file_path, "r") as file:
+                json_data = json.load(file)
+                print("JSON data loaded successfully")
+        except FileNotFoundError as e:
+            print("File not found: ", e)
```

Comment:
Daarnaast is het handig om dingen zoals errata te printen. Als je de try weglaat, dan worden errata vanzelf geprint in de shell